File: app/src/main/java/cn/zjl/datacollector/data/entity/CollectionParameterEntity.java
package cn.zjl.datacollector.data.entity;

import androidx.room.Entity;
import androidx.room.PrimaryKey;

@Entity(tableName = "collection_parameters")
public class CollectionParameterEntity {

    @PrimaryKey(autoGenerate = true)
    public long id;

    public long pointId;

    public float transmitCurrent;

    public int sampleFrequency;

    public int collectionCount;

    public float sampleTime;

    public float electrodeDistance;

    public String transmitterDirection;

    public String customParameters;

    public long createdAt;
}

File: app/src/main/java/cn/zjl/datacollector/data/entity/CollectionParameterEntity.kt
package cn.zjl.datacollector.data.entity

import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity(tableName = "collection_parameters")
data class CollectionParameterEntity(
    @PrimaryKey(autoGenerate = true) var id: Long = 0,
    var pointId: Long = 0,
    var transmitCurrent: Float = 0f,
    var sampleFrequency: Int = 0,
    var collectionCount: Int = 0,
    var sampleTime: Float = 0f,
    var electrodeDistance: Float = 0f,
    var transmitterDirection: String = "",
    var customParameters: String = "",
    var createdAt: Long = 0
)
